flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/InfluxdbReporterTest.java

```diff
@@ -96,33 +96,36 @@ public void testMetricRegistration() throws Exception {
 
 	@Test
 	public void testMetricReporting() throws Exception {
-		MetricRegistryImpl metricRegistry = createMetricRegistry();
+		String retentionPolicy = "one_hour";
+		MetricRegistryImpl metricRegistry = createMetricRegistry(retentionPolicy);
 		try {
 			String metricName = "TestCounter";
 			Counter counter = registerTestMetric(metricName, metricRegistry);
 			counter.inc(42);
 
 			stubFor(post(urlPathEqualTo("/write"))
-				.willReturn(aResponse()
-					.withStatus(200)));
+					.willReturn(aResponse()
+							.withStatus(200)));
```

Comment:
> The code formatting has changed here and 10 lines above.

Reply:
@1u0 - Updated formatting based on your recommendation.  Please let me know if have any more comments.  Thanks again.

Reply:
@1u0, @Myasuka - Please let me know if I can resolve this issue and if there is anything else required for this PR.  Thanks for your advice.

Mans